### DISCUSSION TITLE
Bad Documentation

### DISCUSSION THREAD

**Discussion by txtyash** [twbs/bootstrap#40535] [General]
Being forced to use bootstrap for some reason. I'm coming from tailwind and the documentation on tailwind is absolutely amazing. I can search exactly what values a property can have. I can get entire lists to everything. I just can't seem to find anything  in the bootstrap docs. Maybe in the 2nd or 3rd try...

**Comment by gerardnico** [comment]
You can also too with bootstrap. 

You just need to have your css and JavaScript declared in your ide. If this is what you mean.

Tailwind is a bunch of classes, no? What do you mean with properties you can choose ?

All the best and courage.
